isisd/isisd.h

```diff
@@ -70,7 +70,6 @@ struct isis {
 	struct list *area_list;	/* list of IS-IS areas */
 	struct list *init_circ_list;
 	struct list *nexthops;		  /* IPv4 next hops from this IS */
```

Comment:
If `*nexthops6` is removed. I am guessing `*nexthops` is for both v4 and v6?

Reply:
Good catch. I updated the comment now.